apps/damap-frontend/src/app/components/layout/layout.component.css

```diff
@@ -35,7 +35,7 @@ mat-nav-list {
 }
 
 .mm-navbars-top {
-  background-color: var(--main-color);
+  background-color: #006699;
```

Comment:
Is this required for the update? Might lose customization possibility here (same for the change further down the file)